bin/.preload.sh

```diff
@@ -7,7 +7,7 @@ DFAULTCONFIG="conf/config.properties"
 CUSTOMCONFIG="data/settings/customized_config.properties"
 LOGCONFIG="conf/logs/log-to-file.properties"
 STARTUPFILE="data/startup.tmp"
-DFAULTXmx="-Xmx800m";
+DFAULTXmx="1024m";
```

Comment:
@yukiisbored DFAULTXmx is concatinating with string "-Xmx" to become command line argument of java. here -> https://github.com/loklak/loklak_server/blob/development/bin/.preload.sh#L48